### PR TITLE
Update release notes for 0.119

### PR DESCRIPTION
**Pull request by haozhun**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.119.rst [hunk 1]
```diff
@@ -40,6 +44,8 @@ CLI Changes
 * Skip printing query URL if terminal is too narrow.
 * Allow performing a partial query cancel using ``ctrl-P``.
 * Allow toggling debug mode during query by pressing ``D``.
+* Fix handling of query abortion after result has been partially received.
+* Fix handling of ``ctrl-C`` when displaying results without a pager
```
Comment (cberner) [inline presto-docs/src/main/sphinx/release/release-0.119.rst]: missing period at end

File: presto-docs/src/main/sphinx/release/release-0.119.rst [hunk 2]
```diff
@@ -20,6 +20,10 @@ General Changes
   independent options: ``task.http-response-threads`` and ``task.http-timeout-threads``.
 * Improve handling of negated expressions in join criteria.
 * Add support for Cassandra BLOB types.
+* Fix :func:`arbitrary`, :func:`max_by`, and :func:`min_by` functions when
```
Comment (cberner) [inline presto-docs/src/main/sphinx/release/release-0.119.rst]: I would say "...functions when used with an array, map, or row type"

**Comment by cberner** [comment]
looks good

File: presto-docs/src/main/sphinx/release/release-0.119.rst (hunk 2 above)
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.119.rst]: No comma before "and"

File: presto-docs/src/main/sphinx/release/release-0.119.rst [hunk 3]
```diff
@@ -20,6 +20,10 @@ General Changes
...
+  arguments contain array, map, or row type.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.119.rst]: No comma before "or"

File: presto-docs/src/main/sphinx/release/release-0.119.rst [hunk 4]
```diff
@@ -20,6 +20,10 @@ General Changes
...
+* Fix union coercion when a constant or column appears more than once on the
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.119.rst]: Should this be "when the same constant or"

**Pull request merged**